File: insight/src/test/testData/python/SimplifyBranch.py
def simplifyBranch():
    if True:
        print(True)  # sleep 100ms

    print(False)  # sleep 100ms


def simplifyBranch2():
    print(True)  # sleep 100ms
    if True:
        print(False)  # sleep 100ms

    print(True)  # sleep 100ms

File: insight/src/test/testData/javascript/SimplifyBranch.js
function simplifyBranch() {
    if (true) {
        console.log(true); //sleep 100ms
    }
    console.log(false); //sleep 100ms
}

function simplifyBranch2() {
    console.log(true); //sleep 100ms
    if (true) {
        console.log(false); //sleep 100ms
    }
    console.log(true); //sleep 100ms
}